kosogor-terraform/src/main/kotlin/tanvd/kosogor/terraform/TerraformDsl.kt

```diff
@@ -24,6 +24,20 @@ class TerraformDsl(var project: Project? = null) {
         config.configure()
     }
 
+    data class Parameters(
+            var init: List<String>? = null,
```

Comment:
Use `Iterable`. You don't need to know anything about this type except it can be iterated.

Reply:
Thank you, all is fixed)